packages/web/src/common/services/audius-backend/AudiusBackend.ts

```diff
@@ -234,8 +234,7 @@ type AudiusBackendParams = {
   onLibsInit: (libs: any) => void
   recaptchaSiteKey: Maybe<string>
   recordAnalytics: (
-    event: string,
-    properties?: Record<string, any>,
+    event: { eventName: string; properties?: Record<string, any> },
```

Comment:
do we want to use `Record<string, unknown>` i think that is technically the best version of "any object"

Reply:
Yup, can actually use `AnalyticsEvent` here